vite.config.ts

```diff
@@ -1,8 +1,54 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import { VitePWA, VitePWAOptions } from "vite-plugin-pwa";
+
+const manifestForPlugin: Partial<VitePWAOptions> = {
+  registerType: "autoUpdate",
+  includeAssets: ["favicon.ico"],
+  manifest: {
```

Comment:
can we pull this from the existing manifest.json file using an import?

Reply:
Tried pulling it via import but didn't work...